<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>725</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>990</width>
    <height>725</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>990</width>
    <height>725</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modo de Visualização</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #222433;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>80</y>
      <width>646</width>
      <height>46</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#e62e2e;&quot;&gt;Visualização&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="qt_tipoVisualizacao">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: #9F1823;
border-radius:0px;</string>
       </property>
       <item>
        <property name="text">
         <string>Original</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Perspectiva</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Crop</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cruzetas</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Centroides</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Robôs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Visão</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jogo (Dinâmico)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jogo (Estático)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}</string>
       </property>
       <property name="text">
        <string>Selecionar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnVoltar">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}

</string>
       </property>
       <property name="text">
        <string>Voltar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMenu">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
	color: rgb(255, 255, 255);
	background-color: #9F1823;
	border-radius:0px;
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: #ff0000;
	border-radius:0px;
}

</string>
       </property>
       <property name="text">
        <string>Menu</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="QT_visualizacao">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>140</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
